src/queries.ts

```diff
@@ -102,6 +104,9 @@ function isMultiParams(object: any): object is MultiQueryParams {
 export function canonicalEvents(params: DesktopQueryParams | AndroidQueryParams): string {
   // Needs escaping for regex patterns like '\w' to work (JSON.stringify adds extra unecessary escaping)
   const categories_str = JSON.stringify(params.categories).replace(/\\\\/g, '\\');
+  const always_active_pattern_str = isDesktopParams(params)
+    ? params.always_active_pattern.replace(/\\\\/g, '\\')
```

Comment:
Oops, this replace is probably bad given it's not using `JSON.stringify`. Will fix that in commit directly to master.

Reply:
That's embarrassing xD Thanks for the fix :3